ansible/group_vars/tag_Environment_discovery

```diff
@@ -15,8 +15,6 @@ register_settings:
     enable_register_data_delete: false
     fields_yaml_location: "s3://openregister.discovery.config/fields.yaml"
     registers_yaml_location: "s3://openregister.discovery.config/registers.yaml"
-  address:
```

Comment:
We should still keep this configuration option.

Reply:
@karlbaker02 I thought we only had this enabled because the register was large? Is there another reason?

Reply:
I thought there was some legal issue around address data? I may be mistaken, in which case we could remove it.

Reply:
@karlbaker02 that's possibly true, however, the register only contains 833 records and you could pull all that in one request from the `/records` endpoint within the `page-size` limit anyway.

Reply:
This was only there because the data was large and a download made the app fall over. This won't happen for the small address register in discovery.